world_universities_and_domains.json

```diff
@@ -130977,5 +130980,149 @@
       "uet.edu.al"
     ],
     "country": "Albania"
+  },
+  {
+    "web_pages": [
+      "https://rkgit.edu.in/"
+    ],
+    "name": "Raj Kumar Goel Insititue Of Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "rkgit.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://vbithyd.ac.in/"
+    ],
+    "name": "Vignana Bharathi Institute of science and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "vbithyd.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.ksriet.ac.in/"
+    ],
+    "name": "KSR Institute for engineering and technology",
```

Comment:
I think we can capitalize some words to make them align with others university name
`KSR Institute for Engineering and Technology`

Reply:
Sounds good, I'll change that